src/constants.ts

```diff
@@ -1,4 +1,4 @@
-export const APP_ROOT = 'https://api.wally.xyz/';
+export const APP_ROOT = 'https://api.wally.xyz/v1';
```

Comment:
Are we still not using envars for this?

Reply:
how would we use an env var here? the package gets built before we distribute it, likely as an `npm` package - and won't be run until much later inside of the customer's environment. Should we tell customers to set an env variable when they install the package?

Reply:
🤔 good point. Though I didn't really keep in mind this will be distributed via `npm`. Will you  have to compile down to js for distribution or no? You can ignore this since we're distributing via `npm`. Don't want setting this up to be a hassle to customers. Even though it's expected that they would have some technical ability, the less friction, the better.

Reply:
Yeah - so right now it's getting compiled to js into the `dist/` directory _(and demo/public/sdk/ for the demo app)_ - we might add some other packaging/minification stuff for the public distribution(maybe?) - but I think what's in `dist/` is 90% of the way there